resource/filter.go

```diff
@@ -167,3 +174,57 @@ func (f Filter) matches(r *Resource) bool {
 	}
 	return false
 }
+
+func (c *CreatedTime) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	var v interface{}
+	if err := unmarshal(&v); err != nil {
+		return err
+	}
+	switch value := v.(type) {
+	case time.Time:
+		t, _ := v.(time.Time)
+		*c = CreatedTime{t}
+		return nil
+	case string:
+		d, err := time.ParseDuration(value)
+		if err == nil {
+			*c = CreatedTime{time.Now().UTC().Add(- d)}
+			return nil
+		}
+		var t time.Time
+		err = yaml.Unmarshal([]byte("!!timestamp " + value), &t)
+		if err == nil {
+			*c = CreatedTime{t}
+			return nil
+		}
+		if strings.HasSuffix(value, "d") {
```

Comment:
Ah I see, some more parsing needed as `Duration` only supports `"ns", "us" (or "µs"), "ms", "s", "m", "h"` (https://golang.org/pkg/time/#Duration)

Reply:
Yeah.  Some of the duration units are useful for this, but `d` is not included and way more useful in my case.